src/AzureClient/AzureClient.cs

```diff
@@ -502,6 +502,30 @@ public async Task<ExecutionResult> GetJobListAsync(IChannel channel, string filt
             return jobs.ToExecutionResult();
         }
 
+        public async Task<ExecutionResult> GetQuotaListAsync(IChannel channel)
+        {
+            if (ActiveWorkspace == null)
+            {
+                channel.Stderr($"Please call {GetCommandDisplayName("connect")} before reading quota information.");
+                return AzureClientError.NotConnected.ToExecutionResult();
+            }
+
+            var connectionResult = await RefreshConnectionAsync(channel);
+            if (connectionResult.Status != ExecuteStatus.Ok)
+            {
+                return connectionResult;
+            }
+
+            var quotas = await ActiveWorkspace.ListQuotasAsync() ?? new List<QuotaInfo>();
+
+            if (quotas.Count() == 0)
+            {
+                channel.Stderr("No quota information found in current Azure Quantum workspace.");
```

Comment:
This may not be an error condition, as it correctly reports that there are no quotas to list?

Reply:
You're right. Thanks for the catch!